### ISSUE TITLE
CPS3 games doesn´t work

### ISSUE THREAD

**Issue by Oberworld** [goweiwen/Allium#33]
When you try to launch a Cps3 game, you get a black screen and after some time goes back to game list.

With other CPS3 games the black screen doesn't go back to gamelist and you need to power off the device, extract the sdcard, and insert it in a PC to delete the ".allium\state\current_game" file to be able to boot into the menu again. If not, any time you turn on the console will boot to the black screen.
If you try to launch the games with fbneo you obtain a "failed initializing driver" error.

My romset was working fine in onion, but I prefeer Allium by far. I hope this can be fixed in future releases.

Thank you for your great work, and sorry for my poor english. My best regards.

**Comment by goweiwen** [comment]
Hi, have you tried using different cores? No guarantees that this will work, but it's worth a try. I personally haven't tried any CPS3 games so I don't know which ones will work.

You can press "select" before launching the game to set which core to launch with.

It might also be a lack of memory as Allium doesn't use swap memory. I can look into selectively enabling swap for specific cores or systems if that's the issue.

**Comment by Oberworld** [comment]
Yes, I have tried with all possible cores but it still doesn't work.
The best core for CPS3 it's fbalpha2012_CPS3 but doesn't work.
You can also use fbneo but doesn't work too, but almost this core gives you an error, "failed initializing driver".
after a bit of googling, I see that it's possible that can be a lack of memory, as you can see here:
https://neo-source.com/index.php?topic=3871.msg32760#msg32760

Thank you for answering.

**Comment by Oberworld** [comment]
I have noticed that the same thing happens with PGM (PolyGame Master) games, these games run well on onion, so I definitely think we need swap memory, at least on the fba2012 and fbneo cores.

**Comment by goweiwen** [comment]
I'll enable swap for the FinalBurn cores and mGBA. Please let me know if there are any other cores that require swap.

**Comment by Oberworld** [comment]
Thank you, I'll stay tuned.

**Comment by Oberworld** [comment]
I think that PCSX ReARMed core could benefit of swap memory, specially for more demandong games.
Thant you.
